hack/heartbeat_account.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import httplib
+import urllib
+import os
+import time
+
+BOSKOS_HOST=os.environ.get("BOSKOS_HOST", "boskos")
+BOSKOS_RESOURCE_NAME=os.environ['BOSKOS_RESOURCE_NAME']
```

Comment:
Is anything actually setting `BOSKOS_RESOURCE_NAME`?

Reply:
After some digging, it looks like this also affects the `hack/checkin_account.py` script, where I think we also have the same issue.

Reply:
yes, it's easy to miss. we print the variable in the python script `checkout_account.py` and then source that stdout in:
https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/scripts/ci-conformance.sh#L52

Reply:
then both checkout and heartbeat scripts get it from the env var.